.github/workflows/ci-prover-reusable.yml

```diff
@@ -3,6 +3,34 @@ on:
   workflow_call:
 
 jobs:
```

Comment:
Where do you change the CI to start building prover with "gpu" feature?

Reply:
It's not here yet -- I think we should add it in a separate PR

Reply:
On a second thought, there are no tests there, and the prover is built in the "Build prover images" CI